app/background.cc

```diff
@@ -183,11 +183,14 @@ int grab_background(std::shared_ptr<background_t> pbkd, int width, int height, c
     if (pbkd->video) {
         // grab frame & frame no. under mutex
         std::unique_lock<std::mutex> hold(pbkd->rawmux);
-        cv::resize(pbkd->raw, out, cv::Size(width, height));
+	cv::Rect_<int> crop = calcCropping(pbkd->raw.cols,pbkd->raw.rows,width, height);
+        cv::resize(pbkd->raw(crop), out, cv::Size(width, height));
         frm = pbkd->frame;
     } else {
         // resize still image as requested into out
-        cv::resize(pbkd->raw, out, cv::Size(width, height));
+	cv::Rect_<int> crop = calcCropping(pbkd->raw.cols,pbkd->raw.rows,width, height);
+	cv::resize(pbkd->raw(crop), out, cv::Size(width, height));
+	out = pbkd->raw;
```

Comment:
```suggestion
        cv::Rect_<int> crop = calcCropping(pbkd->raw.cols, pbkd->raw.rows, width, height);
        cv::resize(pbkd->raw(crop), out, cv::Size(width, height));
        out = pbkd->raw;
```

Sure it's `cv::Rect_` instead of `cv::Rect`? Can't test right now.